warrior/WarriorCore/Classes/war_cli_class.py

```diff
@@ -469,12 +469,20 @@ def parser(cls, arglist):
             help="generate the sample cli/netconf/snmp testcase based on "\
             "user provide type")
 
+        warrior_arg.add_argument('-stage_name', action='store', type=str, default=None,\
```

Comment:
what is the expected behaviour if only 'stage_name' or 'kafka_system' is provided, do we need to provide both?